### ISSUE TITLE
Clarification in Coding Standard

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3489]
|<img alt="kehander1's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| kehander1 reported|
|-|-|
|Trac ID|trac#3489|
|Type|defect|
|Created|2008-10-22 22:47:16Z|
|Branch|https://github.com/twisted/twisted/tree/coding-testing-standard-updates-3489|

The link to testing-standard.xhtml is much too easy to miss, and the method of actually running the tests is probably important enough to merit explicit description.  (Also, the word "tests" is repeated twice.)

Attachments:

* [coding-standard-fix.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/dc2/dc2caaad045226aa7bfd10cf40c999c3d66ad6d0/da590b1cf7cb08c286e9c6ae5d66f60a5fc2e4c8.diff) (1947 bytes) - added by kehander1 on 2008-10-22 22:47:36Z - Adds clarification to coding-standard

<details><summary>Searchable metadata</summary>

```
trac-id__3489 3489
type__defect defect
reporter__kehander1 kehander1
priority__normal normal
milestone__ 
branch__branches_coding_testing_standard_updates_3489 branches/coding-testing-standard-updates-3489
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1224715636000000 1224715636000000
changetime__1224784632000000 1224784632000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [25040]) Branching to 'coding-testing-standard-updates-3489'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [25041]) update some old text about testing and twisted

refs [#3489](https://github.com/twisted/twisted/issues/3489)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

I applied the patch and made a bunch more changes.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

Can you also fix the 'shuriken' typo that was introduced yesterday? Other than that it looks good to me and can be merged.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

(In [25042]) zomg rofl learn to spell 手裏剣 gaijin

refs [#3489](https://github.com/twisted/twisted/issues/3489)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [25043]) Merge coding-testing-standard-updates-3489

Author: kehander1, exarkun
Reviewer: thijs
Fixes: [#3489](https://github.com/twisted/twisted/issues/3489)

Update the testing section of the coding standard to talk about:

  * test driven development
  * always working trunk
  * where the test suite is
  * how to run the test suite

Also fix the spelling of a word in one of the existing examples
about docstring conventions (totally unrelated to the rest of the
changes in this merge).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
